build/Common.prod.props

```diff
@@ -59,6 +59,19 @@
     <Message Importance="high" Text="**AssemblyVersionDebug** TargetFramework: $(TargetFramework), MinVerVersion: $(MinVerVersion), MinVerMajor: $(MinVerMajor), MinVerMinor: $(MinVerMinor), MinVerPatch: $(MinVerPatch), MinVerPreRelease: $(MinVerPreRelease), BuildNumber: $(BuildNumber), FileVersion: $(FileVersion), ExposeExperimentalFeatures: $(ExposeExperimentalFeatures)" />
   </Target>
 
+  <Target Name="ResolveExposeExperimentalFeatures" AfterTargets="MinVer">
```

Comment:
Shouldn't this be run after `AssemblyVersionTarget`?

```suggestion
  <Target Name="ResolveExposeExperimentalFeatures" AfterTargets="AssemblyVersionTarget">
```

Reply:
I had that at first but `AssemblyVersionTarget` gets skipped for non-publish builds and we want this section to always run.

Reply:
When publishing builds, is the ordering guaranteed? We don't want this target to be evaluated before `AssemblyVersionTarget`.

Reply:
Not 100% sure about that, but I changed it so it now has a `DependsOnTargets="AssemblyVersionTarget"`